bin/tanner

```diff
@@ -17,7 +17,9 @@ def main():
     """)
     parser = argparse.ArgumentParser()
     parser.add_argument("--config", help="tanner config")
+    parser.add_argument("--base-dir", help="tanner base directory", default='/opt/tanner')
```

Comment:
Newline

Reply:
Add an empty line after this just like we had before.

Reply:
done :)